applications/RomApplication/python_scripts/rom_analysis.py

```diff
@@ -19,9 +19,17 @@ def __init__(self,global_model, parameters):
 
         def _CreateSolver(self):
             """ Create the Solver (and create and import the ModelPart if it is not alread in the model) """
+            self.rom_basis_output_name = 'RomParameters.json' #Default
+            if self.project_parameters.Has("output_processes"):
+                for name, _ in self.project_parameters["output_processes"].items():
+                    if name=="rom_output":
+                        rom_output_paramaters = self.project_parameters["output_processes"]["rom_output"]
+                        rom_basis_output_format = rom_output_paramaters[0]["Parameters"]["rom_basis_output_format"].GetString()
+                        rom_basis_output_name = rom_output_paramaters[0]["Parameters"]["rom_basis_output_name"].GetString()
+                        self.rom_basis_output_name = rom_basis_output_name + "." + rom_basis_output_format
```

Comment:
Why is this related to the `CoSimulationApplication` enabling? I think we already upgrade the output stuf in @Rbravo555 's last PR.

Reply:
This code enables users to have two different `RomParameters.json` files, such as `RomParameters_simulation_1.json` and `RomParameters_simulation_2.json`. This feature can be useful when users need to simulate different scenarios that require different parameters. For example, in the context of potential flow, users may need to simulate both the primal and adjoint problems, which would require different sets of parameters.

Reply:
OK, but please create a dedicated PR for it.